.github/workflows/phpunit.yml

```diff
@@ -23,6 +23,8 @@ jobs:
                       testbench: 5.*
                     - laravel: 6.*
                       testbench: 4.*
+                    - php: 8.0
+                      mockery: "mockery/mockery:^1.4.1"
```

Comment:
Why is `mockery` needed here?

Reply:
It's a dev dependency of laravel/framework.

Reply:
Then you could do like we are doing for `testbench`. You can add it to the dev dependency with a `^1.3.1 | ^1.4.1` version.

And then add it to the matrix that you want php-8 with mockery 1.4.1. A bit like what you did here.

Reply:
I thought it would be clever the way I did it, because there will be multiple choices. l6,p8 => 1.4.1, l7, p8 => 1.4.1, l6,p<8 => 1.0, l7,p<8 => 1.3.1. In the current way, the variable is empty (so it's whatever is required by the dependency) and is only defined, if php8 is used.

Reply:
But if mockery is required as a dev dependency don't we need to also install it when developing?

Reply:
It will be installed automatically as a dev-dependency of laravel/framework. They do allow `^1.3.1 || ^1.4.1`. But as mentioned above, ^1.3 and any other version < 1.4 of mockery does not limit the possible php version to < 8 and therefore, prefer-lowest does install version 1.0, or 1.3 (depending on which laravel/framework is installed). To avoid that, I only install ^1.4.1 explicitly, if we are on php8, just to avoid "prefer-lowest" in that case. But in any other cases I don't want to bother with mockery as we do not need it ourself. If we would not force it to ^1.4.1, the unit tests will fail due to not php 8 compatible code within mockery.

Reply:
Just as a reference, here is one of the failing test runs: https://github.com/Elhebert/laravel-sri/runs/2183803347

Reply:
Make sense then. 👌